src/routes/categories/containers/__tests__/__snapshots__/CategoriesPage.spec.js.snap

```diff
@@ -7,27 +7,27 @@ exports[`CategoriesPage should render CategoryTiles if the path is the root cate
       Array [
         LocationModel {
           "_code": "augsburg",
-          "_eventsEnabled": undefined,
-          "_extrasEnabled": undefined,
-          "_live": undefined,
+          "_eventsEnabled": false,
+          "_extrasEnabled": false,
+          "_live": false,
```

Comment:
does not really matter but should be true

Reply:
These are now the default values defined in CategoryModel. I think undefined makes never sense so I changed it to some boolean.

Reply:
as I said does not really matter, but categories always have to be live (otherwise they are filtered by categories endpoint), so maybe for specification reasons which our tests should be it should be true here

Reply:
I think false is a more reasonable, more protective default. It is true that a non-live location usually won't be rendered. I could override the default value in CategoriesPage.spec.js. What do you think?

Reply:
in this specific case I would prefer true as default value (we could even remove the live attribut completely, because we don't have any non-live categories in our whole app, except in the endpoint, so we could filter categories from the endpoint before mapping them to CategoryModel). eventsEnabled and extrasEnabled should stay false, of course

Reply:
Uh uh uh, I like removing!

Reply:
You mean Location model?

Reply:
right, my bad, than leave it :D

Reply:
but I would change it here to true anyway

Reply:
@Schedulaar what do you think would be a sane default? In this case is really does not matter if its true or false

Reply:
Another option would be that we remove all default values from models and have to specify every value.
If the backend does not deliver some key-value pairs then our models would have default values. This is not an expected behaviour. You would expect an exception.

Reply:
I actually think we shouldn't have defaults there :D

Reply:
So yes, rather raise an exception

Reply:
I created task: https://integreat.atlassian.net/browse/WEBAPP-173